test/vsp/testBsm/runTest.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 source /opt/ros/foxy/setup.bash
-source ../../../other/install/setup.bash
+#source ../../../other/install/setup.bash
+source /home/yilin/SIP_project/carma-msgs/install/setup.bash 
```

Comment:
Lets remove this. I think as long as you source the built transceiver bash you should be ok. We want to refrain from using paths that would break in someone else's environment